src/main/webapp/mentors.js

```diff
@@ -55,81 +105,173 @@ function createMentorElement(mentor) {
+ * @param {String} name
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
+ */
+function addName(name, mentorCardBody) {
+  if (name == null) {
+    return mentorCardBody;
+  }
+  const mentorName = document.createElement('h5');
+  mentorName.className = 'card-title text-primary';
+  mentorName.innerHTML = name;
+  mentorCardBody.appendChild(mentorName);
+  return mentorCardBody;
 }
 
 /**
  * Creates a github profile button element from username.
  * @param {String} gitHubID
- * @return {HTMLElement} userGithubButton
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
  */
-function createGitHubLink(gitHubID) {
+function createGitHubLink(gitHubID, mentorCardBody) {
+  if (gitHubID == null) {
+    return mentorCardBody;
+  }
   const gitHubBaseUrl = 'https://github.com/';
   const gitLink = gitHubBaseUrl.concat(gitHubID);
   const userGithubButton = document.createElement('a');
   userGithubButton.innerText = 'GitHub Profile';
   userGithubButton.className = 'btn btn-primary';
   userGithubButton.role = 'button';
   userGithubButton.href = gitLink;
-  return userGithubButton;
+  mentorCardBody.appendChild(userGithubButton);
+  return mentorCardBody;
 }
 
 /**
  * Creates a block of interest tags.
  * @param {String[]} interestTags
- * @return {HTMLElement} tagDiv
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
  */
-function createTagDiv(interestTags) {
+function createTagDiv(interestTags, mentorCardBody) {
+  if (interestTags == null) {
+    return mentorCardBody;
+  }
   const tagDiv = document.createElement('div');
-  tagDiv.className = 'row p-3';
+  tagDiv.className = 'd-flex justify-content-center p-3';
   for (const tagText of interestTags) {
     const tagElement = document.createElement('div');
-    tagElement.className = 'border border-muted text-muted mr-1 mb-1 badge';
+    tagElement.className = 'border border-muted text-muted mr-1 mb-1 badge text-center';
     tagElement.innerText = tagText;
     tagDiv.append(tagElement);
   }
-  return tagDiv;
+  mentorCardBody.appendChild(tagDiv);
+  return mentorCardBody;
 }
 
 /**
  * Creates a block of project cards.
  * @param {String[]} projectIDs
- * @return {HTMLElement} projectsDiv
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
  */
-function createProjectsDiv(projectIDs) {
+function createProjectsDiv(projectIDs, mentorCardBody) {
+  if (projectIDs == null) {
+    return mentorCardBody;
+  }
   const projectsDiv = document.createElement('div');
-  projectsDiv.className = 'row p-3';
+  projectsDiv.className = 'd-flex justify-content-center p-3';
   for (const projectID of projectIDs) {
     const projectElement = document.createElement('div');
-    projectElement.className = 'card container card-holder col-12' +
-        ' text-center project-card p-3 m-3';
+    projectElement.className = 'card-holder border border-muted text-muted mr-1 mb-1 badge text-center';
```

Comment:
Another line length fix
```suggestion
    projectElement.className = 
        'card-holder border border-muted text-muted mr-1 mb-1 badge text-center';
```